apps/src/music/views/MusicView.jsx

```diff
@@ -885,6 +907,13 @@ class UnconnectedMusicView extends React.Component {
           analyticsReporter={this.analyticsReporter}
           blocklyWorkspace={this.musicBlocklyWorkspace}
           exemplarPlaybackEvents={this.getExemplarPlaybackEvents()}
+          executeCode={
+            AppConfig.getValue('js-editor') === 'true' &&
+            (code => {
+              this.props.setCurrentCode(code);
+              this.executeCompiledSong();
```

Comment:
Thoughts on just creating a separate function that takes the code as a parameter directly (e.g. `this.executeCode(code: string)` or similar)? That way you don't need to plumb this via redux just to read it back again in `executeCompiledSong`

Reply:
Good idea.  Built a new function.